db/migrations/postgres/000026_create_tokenaccount_table.up.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
+CREATE TABLE tokenaccount (
+  seq            SERIAL          PRIMARY KEY,
+  namespace      VARCHAR(64)     NOT NULL,
+  pool_id        UUID            NOT NULL,
+  token_index    VARCHAR(1024)   NOT NULL,
+  identity       VARCHAR(1024)   NOT NULL,
```

Comment:
Just a note that we need to reconcile this identity with the output of https://github.com/hyperledger-labs/firefly/issues/187 before the Tokens work can be considered MVP ready.

Reply:
Yes, the author/identity stuff already merged for token pools is also woefully incomplete, and it'll all need an overhaul when identities are settled in general.